src/Swashbuckle.AspNetCore.Filters/Examples/MvcOutputFormatter.cs

```diff
@@ -48,9 +49,17 @@ private OutputFormatterSelector OutputFormatterSelector
         }
 
         public MvcOutputFormatter(IOptions<MvcOptions> options, ILoggerFactory loggerFactory)

```

Comment:
Could you remove this constructor, so that your new constructor which takes an IServiceProvider is the only constructor?